examples/book-shelf/src/schema.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  CoMap,
+  co,
+  CoList,
+  Account,
+  Profile,
+  Group,
+  ImageDefinition,
+} from "jazz-tools";
+
+export class BookReview extends CoMap {
+  title = co.string;
+  author = co.string;
+  rating = co.number;
+  dateRead? = co.number;
+  review? = co.string;
+  cover? = co.ref(ImageDefinition);
+}
+
+export class ListOfBookReviews extends CoList.Of(co.ref(BookReview)) {}
+
+/** The account root is an app-specific per-user private `CoMap`
+ *  where you can store top-level objects for that user */
+export class AccountRoot extends CoMap {
+  bookReviews = co.ref(ListOfBookReviews);
+}
+
+export class JazzAccount extends Account {
+  profile = co.ref(Profile);
+  root = co.ref(AccountRoot);
+
+  /** The account migration is run on account creation and on every log-in.
+   *  You can use it to set up the account root and any other initial CoValues you need.
+   */
+  migrate(this: JazzAccount, creationProps?: { name: string }) {
+    super.migrate(creationProps);
+    const group = Group.create({ owner: this });
+    group.addMember("everyone", "reader");
+
+    if (!this._refs.root) {
+      this.root = AccountRoot.create(
+        {
+          bookReviews: ListOfBookReviews.create([], { owner: group }),
```

Comment:
When logged in as user A, is it possible to access user B's bookReviews?

I'd like to make a user profile (see `user/[slug]/page.tsx`) to show user's books publicly.

Reply:
Just like you can subclass a custom AccountRoot you can also subclass Profile and add more fields to it which will actually be visible to everyone. Your group setup is correct but the root itself is private.


The profile is public by default and you can even re-use its group by setting the bookReviews owner to me.profile._owner

Reply:
This works. Thank you!